src/components/Form/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Button = ({
+  icon = "",
+  text = "Click me!",
+  spacingModifierClass = "mr-2 mb-2",
+  linkPath,
+  size = "btn-lg",
+}) => (
+  <div className={spacingModifierClass}>
```

Comment:
Wondering if this will always be specific to spacing in the future. Maybe it becomes `modifierClasses`.

Reply:
I agree. No reason to lock spacing in the name. I modified this so it is read as any type of class instead of just spacing.